common/data_refinery_common/migrations/0043_processorjob_abort.py

```diff
@@ -0,0 +1,60 @@
+from django.db import migrations, models
+
+
+def batch_update(queryset, batch=1000, **changes):
+    """ Update per batch """
+    is_done = False
+    current = 0
+
+    while not is_done:
+        current = current + 1
+        start = (current - 1) * batch
```

Comment:
This would be a bit simpler if you incremented `current` at the end of the loop instead of the start because then you wouldn't have to decrement it for this operation.